plugins/com.google.cloud.tools.eclipse.appengine.newproject.maven/src/com/google/cloud/tools/eclipse/appengine/newproject/maven/MavenArchetypeProjectWizard.java

```diff
@@ -40,12 +42,15 @@
 import org.eclipse.jface.wizard.Wizard;
 import org.eclipse.ui.INewWizard;
 import org.eclipse.ui.IWorkbench;
+import org.eclipse.ui.wizards.newresource.BasicNewProjectResourceWizard;
 
-public class MavenArchetypeProjectWizard extends Wizard implements INewWizard {
+public class MavenArchetypeProjectWizard extends Wizard
```

Comment:
Question: if we change `plugin.xml` to use `com.google.cloud.tools.eclipse.util.service.ServiceContextFactory` like in the non-Maven `plugin.xml`, `configElement` could be injected, we won't need `setInitializationData()`, and this class will not implement `IExecutableExtension`?